ydb/public/lib/ydb_cli/commands/interactive/line_reader.cpp

```diff
@@ -34,63 +36,50 @@ std::optional<FileHandlerLockGuard> LockFile(TFileHandle & fileHandle) {
     return FileHandlerLockGuard(&fileHandle);
 }
 
+replxx::Replxx::Color ReplxxColorOf(ECandidateKind kind) {
+    const auto schema = YQLHighlight::ColorSchema::Monaco();
+    switch (kind) {
+    case ECandidateKind::Keyword:
+      return schema.keyword;
+    }
+}
```

Comment:
Here I realized that it would be nice to extract color schema to separate from highlighting unit.